pkgs/development/r-modules/default.nix

```
@@ -657,7 +657,7 @@ let
     rGEDI = with pkgs; [ libgeotiff.dev libaec zlib.dev hdf5.dev ];
     rawrr = [ pkgs.mono ];
     HDF5Array = [ pkgs.zlib.dev ];
-    FLAMES = [ pkgs.zlib.dev ];
+    FLAMES = with pkgs; [ zlib.dev bzip2.dev xz.dev ];
```

```suggestion
    FLAMES = [ pkgs.bzip2.dev pkgs.xz.dev pkgs.zlib.dev ];
```
Avoid `with` whenever you can. Can lead to issues in static analysis.